data-model/SDP-sample-records/textual-artifact_sample1.json

```diff
@@ -0,0 +1,65 @@
+{
+  "id": "ark:/21198/adf20c1f",
+  "type": "Textual Artifact",
+  "label": "Sinai Syriac 2A, ff. 1-152v, Gospels",
+  "unit_start": {
+    "parent_id": "ark:/21198/z1x07bdf",
```

Comment:
for textual artifacts I'm not sure if we want to use the ms object ID (since it matches the shelfmark in the label field and is most securely associated with the folio number) or the codicological unit Id (as it is the direct parent object, and may span multiple manuscript objects)? Currently I've used the ms object ID, but we should discuss